bin/test_runner.rb

```diff
@@ -129,12 +131,17 @@ def wait
   private
 
   def cloud_name(name)
+    # The rocket monkey naming convention goes like this:
+    # prefix_cloud_region_testname...
+    # What this test calls cloud name is actually 'cloud_region'
+    # So split on _ and get those two, for example.
+    # rl10lin_Google_Silicon_monitoring...
+    # yields 'Google_Silicon' as cloud name.
     name.split('_')[1..2].join('_')
   end
 
   def total_deps_up(filter)
-    deployments = @rsclient.deployments.index(filter: ["name==#{filter}"])
-    deployments.length
+    deployments = @rsclient.deployments.index(filter: ["name==#{filter}"]).length
```

Comment:
I was on the fence on this one for a bit.   We could use a style guide.  One was proposed on another team on our wookie, but I don't think it was adopted... the one proposed is small and doesn't really say much on this... https://github.com/styleguide/ruby   

then again, it was a direct search and replace... so ...done